Library/Homebrew/cmd/log.rb

```diff
@@ -22,10 +23,10 @@ def git_log(cd_dir, path = nil, tap = nil)
     repo = Utils.popen_read("git rev-parse --show-toplevel").chomp
     if tap
       name = tap.to_s
-      git_cd = "$(brew --repo #{tap})"
+      git_cd = "$(brew --repo #{tap})".for_shell
     elsif cd_dir == HOMEBREW_REPOSITORY
       name = "Homebrew/brew"
-      git_cd = "$(brew --repo)"
+      git_cd = "$(brew --repo)".for_shell
```

Comment:
There's a problem here. This is used below with surrounding quotes, however `"(brew --repo)"` in `fish` will just be a string, not a subshell.

Reply:
in `fish` it gets translated to `(brew --repo)` which should still be correct for the fish-shell.

Reply:
Look at the usage below, which is `"#{git_cd}"`.

Reply:
ah, I see this issue.  Let me investigate....

Reply:
I addressed the issue by removing the enclosing quotes for fish-shell (they're not needed in fish).